configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
I'm going to be adding a feature to Bazel in the next month or so that will allow us to mirror dependencies. So any flakiness an extra dependency adds would only be temporary.